posthog/tasks/process_event.py

```diff
@@ -10,6 +10,19 @@
 from sentry_sdk import capture_exception
 
 from posthog.models import Element, Event, Person, SessionRecordingEvent, Team
+from posthog.models.feature_flag import FeatureFlag
+
+
+def get_active_feature_flags(team: Team, distinct_id: str) -> List[str]:
```

Comment:
Small issue: I think this logic is worth moving to a separate file.

Two reasons:
1. It creates a weird dependency between /decide endpoint as well as process_event.
2. It makes code harder to follow - feature flags won't be relevant most times you're reading process_event, yet it's the first code you need to read through and ~10% of the file

Reply:
Done